### PR TITLE
fix: to txt file output path parent of input path

### LINKED ISSUE #57: svg2txt: passing current directory saves in parent directory
Passing the current folder, results in the extracted colors being saved in the parent directory instead of the current directory.

```
cd ../compute
<redacted>/harmony svg2txt -r -f hexcode ./
Colors extracted and saved to /Users/adrian/Downloads/_Textual/Azure_Public_Service_Icons/Icons/compute.txt
```

Expected behavior was to save the file in the current directory, where the command was run.

### LINKED ISSUE #59: image2txt: passing -r option does not save the result in the folder where command was run
Passing the -r option saves the file in the parent folder of the argument passed as source folder of PNG files.

Expected behavior would be by default to save the result in the folder where the command was run.

### PR DESCRIPTION
**Pull request by henriquekirchheck**
Changed output file path from "image2txt" and "svg2txt" commands to the current working directory instead of parent of input file or folder.

This pull request assumes that you want the name of the directory when the path argument a directory and the name of the file when the path argument is a file.

Fixes #59
Fixes #57

### CODE REVIEW

**Comment by AdrianSimionov** [comment]
Does this require any tests to be added to the CI/CD? If so, please add it to this pull request.

**Comment by henriquekirchheck** [comment]
> Does this require any tests to be added to the CI/CD? If so, please add it to this pull request.

While all the current tests pass, I could add more tests to test the exact behavior that I made it have. But I would like confirmation that the behavior implemented is the correct one. I'll provide a example of how it would function.

If the command `harmony image2txt -r ~/foo` is run in the folder `/home/user/bar`, the output file should reside in `/home/user/bar/foo.txt`.

If the command `harmony image2txt ~/foo/baz.png` is run in the folder `/home/user/bar`, the output file should reside in `/home/user/bar/baz.txt`

**Comment by AdrianSimionov** [comment]
I believe the behavior you described is right.

Is the behavior you described captured in the documentation? If not, great if it would be added.

**Comment by henriquekirchheck** [comment]
Could you guide me to where documentation is located, the only one I could find was Spec for the Apple CLR Color Palette Format

File: harmony/commands.py [hunk 1]
```diff
@@ -73,8 +74,14 @@ def convert_to_txt_file(self, path: pathlib.Path) -> None:
         colors: Tuple[core.Color, ...] = core_services.extract_colors_from_path(
             path, self.__reading_strategy, self.__recursively
         )
+        final_file_basename = (
```
Comment (xlurio) [inline harmony/commands.py:77]: The "CommandUtils" classes exist only for avoid duplication between the commands, they should not have business logic. Please, move the new logic to a new method in "PathGenerator" class and call it here

**Review by xlurio — CHANGES_REQUESTED** [review]

**Comment by xlurio** [comment]
> Could you guide me to where documentation is located? The only one I could find was Spec for the Apple CLR Color Palette Format

There is an issue to move all the documentation to the `README.md` file, but for now it is located at https://github.com/xlurio/harmony-docs

**Comment by xlurio** [comment]
Welcome to the team, Henrique!

**Comment by henriquekirchheck** [comment]
> Welcome to the team, Henrique!

Thank you!

I have written tests for the new behavior, I will move the logic to PathGenerator asap

**Pull request merged**